core/store/migrate/migrations/0128_add_job_gas_limit.sql

```diff
@@ -0,0 +1,4 @@
+-- +goose Up
+ALTER TABLE jobs ADD COLUMN gas_limit INTEGER DEFAULT NULL;
```

Comment:
Note that `uint32` can accomodate values too large to fit inside a postgres `integer`. You may wish to consider using `BIGINT` here

Reply:
Fixed